pkg/webhooks/admission/hypernodes/validate/admit_hypernode.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2024 The Volcano Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validate
+
+import (
+	"fmt"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	whv1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/klog/v2"
+
+	hypernodev1alpha1 "volcano.sh/apis/pkg/apis/hypernode/v1alpha1"
```

Comment:
should be hypernodev1alpha1 "volcano.sh/apis/pkg/apis/topology/v1alpha1"

Reply:
fixed